.changeset/friendly-parents-guess.md

```diff
@@ -0,0 +1,6 @@
+---
+"@chakra-ui/docs": patch
```

Comment:
Can probably skip adding changesets for `@chakra-ui/docs` in the future. We don't use the version for anything for that, and since the package is `private`, it's meaningless anyway.

Reply:
Okay, noted 👍